### DISCUSSION TITLE
Fulfillment Inbound API v2024-03-20. Operation: createInboundPlan

### DISCUSSION THREAD

**Discussion by KirillChekalin** [amzn/selling-partner-api-models#3875] [General]
Subject: Issue with createInboundPlan API - Unexpected Validation Error

Dear [Support Team],

I am writing to report an issue I encountered while using the createInboundPlan API. Despite following the documentation, I am receiving a validation error that is not addressed in the provided guidelines.

Error Message:

`    "1 validation error detected: Value null at 'request.sourceAddress.phoneNumber' failed to satisfy constraint: Member must not be null"`
    
According to the current documentation, the phoneNumber field for sourceAddress is not listed as a required property. However, the API returns an error indicating that this field must not be null.

I would appreciate your assistance in resolving this issue. Could you please confirm whether the phoneNumber field is indeed required and, if so, update the documentation accordingly? If this is an unexpected validation error, please let me know how to proceed.

Thank you for your support.

**Comment by MojoJake** [comment]
I am also seeing this error. The phone number was not required last week. Also, even when I provide it I still get the error. This appears to have been a change within the last 3-4 days.

**Comment by brandondev1** [comment]
Also getting this error.  Not only is phoneNumber not required, it's not listed as being a valid member property of sourceAddress at all.  Adding a phoneNumber also doesn't work.  It returns the same error regardless of the property being present with a valid value.

**Comment by MojoJake** [comment]
I created a bug for this https://github.com/amzn/selling-partner-api-docs/issues/3778

**Comment by KirillChekalin** [comment]
📢 Update: createInboundPlan API Now Working Without Phone Number
Hi everyone!

I wanted to share an update for those of us using the createInboundPlan API. 🎉

What happened?
Previously, many of us encountered an error when creating an inbound plan, which indicated that the phoneNumber field in the sourceAddress was mandatory. This was unexpected since the documentation did not list this field as required.

Good news!
Today, I discovered that this issue has been resolved. The phone number is no longer required in the request to create an inbound plan, and the API now works as documented.
